docs/client-libraries-cluster-level-failover.md

```diff
@@ -26,7 +26,8 @@ For more information about cluster-level failover, including concepts, benefits,
   * The number of BookKeeper nodes should be equal to or greater than the ensemble quorum.
   * The number of ZooKeeper nodes should be equal to or greater than 3.
 * **Turn on geo-replication** between the primary cluster and any dependent cluster (primary to backup or backup to backup) to prevent data loss.
-* Set `replicateSubscriptionState` to `true` when creating consumers.
+* Set `enableReplicatedSubscriptions` to `true` in [`broker.conf`](https://github.com/apache/pulsar/blob/470b674016c8718f2dfd0a0f93cf02d49af0fead/conf/broker.conf#L592).
```

Comment:
Can we simply say "enable replicated subscription" here with the link pointing to https://pulsar.apache.org/docs/2.11.x/administration-geo/#enable-replicated-subscription which will include both broker configuration and consumer configuration?

Reply:
Good catch! Updated. PTAL.